Horizon/Horizon/Sources/Features/Dashboard/DashboardAssembly.swift

```diff
@@ -19,7 +19,10 @@
 import Foundation
 
 final class DashboardAssembly {
+    static func createGetProgramsInteractor() -> GetProgramsInteractor {
```

Comment:
To be consistent with other factory methods.
```suggestion
    static func makeGetProgramsInteractor() -> GetProgramsInteractor {
```